experiments/tf_trainer/keras_gru_attention/run.py

```diff
@@ -23,8 +23,6 @@
 tf.app.flags.DEFINE_string("embeddings_path",
                            "local_data/glove.6B/glove.6B.100d.txt",
                            "Path to the embeddings file.")
-tf.app.flags.DEFINE_boolean("is_binary_embedding", False,
```

Comment:
Should we remove this from all run.py (e.g. tf_gru_attention/)?

Reply:
Good point. done.